pre-docs/inventory_file.md

```diff
@@ -123,7 +146,7 @@ The inventory file uses the following objects.
 
     A map of fact names and values. values may include arrays or nested maps.
 
--   **Features**
+-   **Feature**
```

Comment:
I can't figure out why you changed this to singular, because we refer to features (plural) everywhere else on this page.

Reply:
It was specifically related to how this section describes things. We refer to the `features` entry as `Array[Feature]`, and this section describes a single feature.